From 58be2dd9647632c2c49c29c29c889888f0a7c85f Mon Sep 17 00:00:00 2001
From: "F. von Gellhorn" <flinux@vongellhorn.ch>
Date: Sun, 28 Jun 2020 11:43:58 +0200
Subject: [PATCH 1/1] Solus tweaks to the default config

- Decrease desktop icon sizes
- Move showdesktop between systemtray and the clock
---
 containments/desktop/package/contents/config/main.xml           | 2 +-
 .../org.kde.plasma.desktop.defaultPanel/contents/layout.js      | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/containments/desktop/package/contents/config/main.xml b/containments/desktop/package/contents/config/main.xml
index 5703999..8c02e38 100644
--- a/containments/desktop/package/contents/config/main.xml
+++ b/containments/desktop/package/contents/config/main.xml
@@ -100,7 +100,7 @@
     </entry>
     <entry name="iconSize" type="Int">
       <label>The icon size to use for Folder View icons. One of the list of Plasma icon sizes.</label>
-      <default>4</default>
+      <default>3</default>
     </entry>
     <entry name="labelWidth" type="Int">
       <label>The width of the grid cells when using icon view. 0 = Narrow, 1 = Medium, 2 = Wide</label>
diff --git a/layout-templates/org.kde.plasma.desktop.defaultPanel/contents/layout.js b/layout-templates/org.kde.plasma.desktop.defaultPanel/contents/layout.js
index 048671e..7a9a3d5 100644
--- a/layout-templates/org.kde.plasma.desktop.defaultPanel/contents/layout.js
+++ b/layout-templates/org.kde.plasma.desktop.defaultPanel/contents/layout.js
@@ -79,6 +79,6 @@ if (langIds.indexOf(languageId) != -1) {
 }
 
 panel.addWidget("org.kde.plasma.systemtray")
+panel.addWidget("org.kde.plasma.minimizeall")
 panel.addWidget("org.kde.plasma.digitalclock")
-panel.addWidget("org.kde.plasma.showdesktop")
 
-- 
2.27.0

